Tests failing when importing components using absolute path

<!--
IF YOU DON'T FILL OUT THE FOLLOWING INFORMATION WE MIGHT CLOSE YOUR ISSUE WITHOUT INVESTIGATING
-->

**I'm submitting a ...**  (check one with "x")

```
[ ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[X] support request
```

**Current behavior**
Tests fail when absolute path import is used

**Expected behavior**
All tests to pass (as they do when I replace the absolute path with the relative equivalent one).

**Minimal reproduction of the problem with instructions**
Just running tests with some component import. Only when using relative paths the test is able to import them:

```js
import Gradient from '@/components/Gradient'     <--- this does not work
import Gradient from 'src/components/Gradient'   <--- this does not work
import Gradient from '../components/Gradient'    <--- this does
```

Console output:

```bash
 FAIL  src/components/Background/background.test.js
  ● Test suite failed to run

    Cannot find module '@/components/Gradient' from 'Background.vue'

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:179:17)
      at src/components/Background/Background.vue:167:17
      at Object.<anonymous> (src/components/Background/Background.vue:210:3)
```

Obviously, my `import Component from @/components/Whatever` works properly in my app.


Just in case, my jest object config from `package.json`:

```js
  "jest": {
    "moduleFileExtensions": ["js", "vue"],
    "moduleNameMapper": {
      "src/components/([^\\.]*)$": "<rootDir>/src/components/$1.vue",
      "^vue$": "vue/dist/vue.common.js",
      "src/([^\\.]*)$": "<rootDir>/src/$1.vue",
      "(.*)/(.*)$": "$1/$2.vue"
    },
    "snapshotSerializers": [
      "jest-serializer-html"
    ],
    "collectCoverageFrom": [
      "src/**/*.{js,jsx}"
    ],
    "transform": {
      "^.+\\.js$": "<rootDir>/node_modules/babel-jest",
      ".*\\.(vue)$": "<rootDir>/node_modules/jest-vue-preprocessor"
    }
  }
```

Tried adding a new line in ModuleNameMapper: `"@/([^\\.]*)$": "<rootDir>/src/$1.vue",` but did not work. I'm not sure if that even make sense :)

I'm using `vue-cli`, so in `webpack.base.conf.js` there's the `@` alias setup:

```js
...
  resolve: {
    extensions: ['.js', '.vue', '.json'],
    alias: {
      'vue$': 'vue/dist/vue.esm.js',
      '@': resolve('src')
    }
  }
```


**Please tell us about your environment:**

* **jest-vue-preprocessor:** 1.0.1
<!-- Please be specific, e.g. major.minor.patch -->

* **Node version :** 8.1.2 (npm 5.3.0)

* **Platform:** OSX


Thanks for your time!

--- a/README.md
+++ b/README.md
@@ -29,6 +29,9 @@ Portions both preprocessors are heavily based [vueify](https://github.com/vuejs/
   3.  Start writing test that can import `*.vue` components - see example **./tests/index.spec.js**
   4.  Profit!
 
+  ### Customization
+
+  * non-relative component imports - you properly set jest's `moduleMapper` option - see [#25](https://github.com/vire/jest-vue-preprocessor/issues/25)
 
   ### [Contributing guide](https://github.com/vire/jest-vue-preprocessor/blob/master/CONTRIBUTING.md)
 

--- a/package.json
+++ b/package.json
@@ -61,6 +61,9 @@
       "js",
       "vue"
     ],
+    "moduleNameMapper": {
+      "^test/fixtures/(.*)": "<rootDir>/test/fixtures/$1"
+    },
     "transform": {
       "^.+\\.js$": "<rootDir>/node_modules/babel-jest",
       ".*\\.(vue)$": "<rootDir>/index.js"

--- a/test/index.spec.js
+++ b/test/index.spec.js
@@ -2,6 +2,7 @@ import Vue from 'vue';
 import FooComponent from './fixtures/FooComponent.vue';
 import TsComponent from './fixtures/TsComponent.vue';
 import TypescriptComponent from './fixtures/TypescriptComponent.vue';
+import AbsolutePathComponent from 'test/fixtures/FooComponent.vue';
 
 const doTest = (Component) => {
   const vm = new Vue({
@@ -32,5 +33,9 @@ describe('preprocessor', () => {
 
   it('should process a `.vue` file with typescript lang', () => {
     doTest(TypescriptComponent);
+  });
+
+  it('should process an absolute path of a `.vue` Component', () => {
+    doTest(AbsolutePathComponent);
   });
 });

